client/app/components/queries/schema-browser.html

```diff
@@ -1,30 +0,0 @@
-<div class="schema-container" ng-if="$ctrl.schema">
-  <div class="schema-control">
-    <input type="text" placeholder="Search schema..." class="form-control" ng-model="$ctrl.schemaFilter" ng-model-options="{ debounce: 500 }" ng-disabled="$ctrl.isEmpty()" ng-change="$ctrl.splitFilter($ctrl.schemaFilter);">
-    <button class="btn btn-default"
-            title="Refresh Schema"
-            ng-click="$ctrl.onRefresh()">
-      <span class="zmdi zmdi-refresh"></span>
-    </button>
-  </div>
-
-  <div class="schema-browser" vs-repeat vs-size="$ctrl.getSize(table)">
-    <div ng-repeat="table in $ctrl.schema | filter:$ctrl.schemaFilterObject track by table.name">
```

Comment:
This file was the only place that was using `angular-vs-repeat` - consider removing that dependency too.